docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -404,6 +399,51 @@ and BatchScanner passing in the name of the context, app1 in the example above.
 iterators to load classes from the locations defined by the context. Passing the context name to the scanners allows you to override the table setting
 to load only scan time iterators from a different location. 
 
+=== Vendor-specific configuration
```

Comment:
> If vendors release new versions and these docs need to be updated, the user manual can be updated in the Accumulo repo and pushed to the website without creating a new bugfix release.

That addresses part of my concern.  Still does not allow linking to the documentation from accumulo-env.sh.  However, the inability to link between different kinds of Accumulo documentation is a more general issue that should not be addresses here.